mypy/semanal.py

```diff
@@ -68,8 +68,7 @@
 from mypy.types import (
     FunctionLike, UnboundType, TypeVarDef, TupleType, UnionType, StarType, function_type,
     CallableType, Overloaded, Instance, Type, AnyType,
-    TypeTranslator, TypeOfAny, TypeType,
-)
+    TypeTranslator, TypeOfAny, TypeType, NoneTyp)
```

Comment:
Please leave the `)` on a line by itself and add a trailing comma, preserving the original formatting.